frontend/src/components/layout/LayoutCol.svelte

```diff
@@ -53,7 +53,6 @@
 	on:mouseout
 	on:mouseup
 	on:select
-	on:wheel
```

Comment:
Why did these three need to be removed?

Reply:
If you want to use them then you have to mark them as either `|passive` or `|nonpassive` in order to avoid a warning in chromium. This is impossible to do whilst they are in this ugly wrapper component.

Reply:
I see, I wasn't aware of `|passive| and `|nonpassive` in Svelte. For context, the goal was to port `LayoutCol` and `LayoutRow` from Vue with as few syntactic changes as possible, even though Svelte doesn't handle this concept as nicely as Vue did. Maybe someday I'll go and find a better solution for this. But that'll take a while and I need to experiment with approaches until I find one I like best. So this wrapper component concept is definitely not the cleanest but it's acceptable *for the moment* given the constraints.

For this, can we just do two lines for `on:wheel|passive` and `on:wheel|nonpassive` or does it have to be one or the other? Can we pick a default that will almost always be what we want?

Reply:
You have to pick one (as otherwise the event would be forwarded twice). You cannot override your choice. You could choose a default - if you choose passive then no handlers can use preventDefault; if you choose nonpassive then you would never be able to handle scroll passively (which has better performance).

Reply:
Thanks for explaining. I'll have to think about that and dig around in the code tomorrow once I'm not so tired to decide on a default. Or if you can pick one that seems sensible to you (like if you know we have no usages of preventDefault anywhere in the code) then go for that if you'd like. But if in doubt, I can take a closer look tomorrow. Thank you!